src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/FormViewNetworkWideFields.tsx

```diff
@@ -132,24 +131,13 @@ const IpConfigFields: React.FC<{
             bodyContent="An IP address to where any IP packet should be forwarded in case there is no other routing rule configured for a destination IP."
           />
         }
-        name={`${fieldName}.gateway`}
+        name={`${fieldName}.gatewayOcmSelectField`}
```

Comment:
I think this name change is breaking a few things:
- When loading a cluster which has set configuration for Dual-stack static IP, the gateway fields appear empty (there were correctly set and visible previously)
- When filling the form from scratch, it keeps as invalid even though no errors are highlighted but user can't proceed

Reply:
True, i've changed it